awx/main/models/inventory.py

```diff
@@ -428,7 +429,8 @@ def update_computed_fields(self, update_groups=True, update_hosts=True):
         '''
         Update model fields that are computed from database relationships.
         '''
-        logger.debug("Going to update inventory computed fields")
+        logger.debug("Going to update inventory computed fields, pk={0}".format(self.pk))
```

Comment:
The biggest issue here is that we get start / stop logs, but with so much happening, we don't know which start corresponds to which stop. That's why the pk is added here. Otherwise, we shouldn't bother issuing a start log.